Libraries/Components/SafeAreaView/__tests__/SafeAreaView-test.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @emails oncall+react_native
+ * @flow
+ */
+
+'use strict';
+
+const SafeAreaView = require('SafeAreaView');
+
+describe('<SafeAreaView />', () => {
+  it('should set displayName to prevent <Component /> regressions', () => {
```

Comment:
Can you add all 4 of the tests that are in the other PR? They all catch slightly different but related issues. We want the same tests that also render to snapshots.

Reply:
@TheSavior Added!